.gitlab-ci.yml

```
@@ -859,45 +854,39 @@ workflow:
 .on_kitchen_tests_a6:
```

```suggestion
.on_installer_e2e_tests_a6:
```
But this could deserve a dedicated PR I guess, as it requires a modification of this reference as well

---

Yes I would prefer to have a dedicated PR for that. This one change the logic and we could have another that change only the naming